tests/unit_tests/test_game_rating.py

```diff
@@ -984,10 +1047,292 @@ async def test_single_wrong_report_still_rated_correctly(game: Game, player_fact
     winning_ids = log_dict["teams"][str(log_dict["winning_team"])]
     for player_id, new_rating in results.ratings.items():
         if player_id in winning_ids:
-            assert new_rating.mu > old_rating
+            assert new_rating.mean > old_rating
         else:
-            assert new_rating.mu < old_rating
+            assert new_rating.mean < old_rating
+
+
+# NOTE: The following test_rating_adjustment_* tests were created by setting up
+# the inputs and copying the results to the expected assertions. They exist
+# primarily to document the current behaviour and detect bugs in case the
+# behaviour changes, however, there is no reason why the rating initialization
+# couldn't be changed and the values in these tests updated.
+async def do_test_rating_adjustment(
+    rating_service,
+    player_factory,
+    ratings,
+    expected_results
+):
+    team1 = set()
+    team2 = set()
+    teams = (team1, team2)
+    for player_id, ratings in ratings.items():
+        _ = player_factory(player_id=player_id)
+        for rating_type, rating in ratings.items():
+            rating_service.set_mock_rating(player_id, rating_type, rating)
+        # Odds on team1, evens on team2
+        teams[(player_id + 1) % 2].add(player_id)
+
+    summary = GameRatingSummary(
+        game_id=1,
+        rating_type=RatingType.LADDER_1V1,
+        teams=[
+            TeamRatingSummary(GameOutcome.VICTORY, team1, []),
+            TeamRatingSummary(GameOutcome.DEFEAT, team2, [])
+        ]
+    )
+    await rating_service._rate(summary)
+
+    results = get_persisted_results(rating_service)
+
+    assert len(results) == len(expected_results)
+    for result, expected in zip(results, expected_results):
+        assert result.rating_type == expected.rating_type
+        assert result.ratings == expected.ratings
+        team1_outcomes = {id: GameOutcome.VICTORY for id in team1}
+        team2_outcomes = {id: GameOutcome.DEFEAT for id in team2}
+        # TODO: When updating from 3.7 to 3.9, use dict union {} | {}
+        assert result.outcomes == {**team1_outcomes, **team2_outcomes}
+
+
+async def test_rating_adjustment_1v1_newbies(rating_service, player_factory):
+    IGNORED = object()
+
+    # Both players have no ratings yet
+    await do_test_rating_adjustment(
+        rating_service,
+        player_factory,
+        ratings={1: {}, 2: {}},
+        expected_results=[
+            PersistedResults(
+                rating_type=RatingType.LADDER_1V1,
+                ratings={
+                    1: pytest.approx((1766, 429), abs=1),
+                    2: pytest.approx((1235, 429), abs=1),
+                },
+                outcomes=IGNORED
+            ),
+            PersistedResults(
+                rating_type=RatingType.GLOBAL,
+                ratings={1: pytest.approx((1766, 429), abs=1)},
+                outcomes=IGNORED
+            )
+        ]
+    )
+
+
+async def test_rating_adjustment_1v1_ladder_newbies_global_pros(
```

Comment:
Should add a test (or modify this one) where the global is higher than ladder, but still under 1400 displayed rating